scheduler/replica_scheduler.go

```diff
@@ -291,6 +296,11 @@ func (rcs *ReplicaScheduler) filterNodeDisksForReplica(node *longhorn.Node, disk
 	multiError = util.NewMultiError()
 	preferredDisks = map[string]*Disk{}
 
+	tolerateEmptyDiskSelectorVolume, err := rcs.ds.GetSettingAsBool(types.SettingNameTolerateEmptyDiskSelectorVolume)
+	if err != nil {
+		logrus.Errorf("Failed to get tolerate empty tag volume setting: %v", err)
```

Comment:
Make the message exact. Need to return err here? (the same for other places)

`"Failed to get % setting", types.SettingNameTolerateEmptyDiskSelectorVolume`

Reply:
Fixed, and return the error when setting is not found